specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview/managedClusters.json

```diff
@@ -6835,6 +6839,16 @@
       },
       "description": "The security settings of an agent pool."
     },
+    "AgentPoolGPUProfile": {
+      "type": "object",
+      "properties": {
+        "installGPUDriver": {
+          "type": "boolean",
+          "title": "Whether to auto install GPU drivers or not.",
+          "description": "The default value is true. GPU Driver Installation can only be set true when VM has associated GPU resource. It is intended as an opt-out feature for users who want to install GPU drivers themselves to avoid any installations."
```

Comment:
```suggestion
          "description": "The default value is true when the vmSize of the agent pool contains a GPU, false otherwise. GPU Driver Installation can only be set true when VM has an associated GPU resource. Setting this field to false prevents automatic GPU driver installation. In that case, in order for the GPU to be usable, the user must perform GPU driver installation themselves."
```